### PR TITLE
Par 1821 small fixes to dropdown

### PR DESCRIPTION
**Pull request by vetlesolgaard**
- [x] Design review

**Changes**
- There is too much space in the drop-down below the search result
- Make the “X” smaller (1rem)
- Looks like its the wrong font once you’ve selected something

❔**Questions**
- Seems a bit weird that the `Tooltip` component is used for showing search content for the `DropdownFilter`🤔 Is this something we should change?

📸**Screenshot:**
Screenshot 2022-08-23 at 16 30 30

### CODE REVIEW

File: src/Nordea/Components/DropdownFilter.elm [hunk 1]
```diff
@@ -117,7 +116,7 @@ init { onInput, input, onSelect, items } =
         , onSelect = onSelect
         , input = input
         , onFocus = Nothing
-        , hasFocus = True
+        , hasFocus = False
```
Comment (fredrikloberg) [inline src/Nordea/Components/DropdownFilter.elm:119]: Should double check that this doesn't break anything in FF. I think the partnerhub super user login page in FF may rely on this

Comment (vetlesolgaard) [reply src/Nordea/Components/DropdownFilter.elm:119]: Nice spot! It seemed not necessary to have this set to `True`. I'll check it out

File: src/Nordea/Components/DropdownFilter.elm [hunk 2]
```diff
@@ -134,17 +133,43 @@ view attrs (DropdownFilter config) =
...
+            let
+                viewHeader text =
+                    Html.li
+                        [ css
+                            [ color Colors.gray
+                            , margin2 (rem 0) (rem 0.625)
+                            , fontSize (rem 0.75)
+                            , fontWeight (int 400)
+                            , lineHeight (rem 1)
+                            ]
```
Comment (fredrikloberg) [inline src/Nordea/Components/DropdownFilter.elm:145]: Do this need to be a custom styled text, or can we use of the the text components? (and maybe add a few overrides). May be worth discussing the header style with Thea, Andreas has already complained that they are somewhat hard to read with the current style

**Review by fredrikloberg — APPROVED** [review]
Looks good! Regarding the use of Tooltip, I find that it serves this purpose very well. The only thing is maybe the name of it. The news dropdown in PartnerHub also uses the tooltip for its content

File: src/Nordea/Components/DropdownFilter.elm (hunk 1 above)
Comment (vetlesolgaard) [reply src/Nordea/Components/DropdownFilter.elm:119]: It seems to only be in use in the configuration page, and for the PartnerHub page, both seem to work fine with it being `False` 😊

File: src/Nordea/Components/DropdownFilter.elm (hunk 2 above)
Comment (vetlesolgaard) [reply src/Nordea/Components/DropdownFilter.elm:145]: Used the `TextTinyLight` variant for now, seems like the closest to what we want😊 I'll have a talk with Thea a day she is at the office

**Pull request merged**